data/Head1/Head1.monop

```diff
@@ -0,0 +1,6 @@
+0 0 0.4250  1
+0 0 0.6800  1
+0 0 0.7650  1
+0 0 0.8075  1
+0 0 0.8415  1
+0 0 0.8075  1.5
```

Comment:
Comment for the line below. You provided the Head1/Head1.msm, which is most probably the expected result of the test. Can we add some comparison test that checks that the output of the test is indeed that one.
I'm also interested on how this file was obtained. If this is by running the OpenMEEG code, we will just check that the code always provide the same result. If the values were obtained with another numerical model, then we might check that the obtained values are indeed those that are expected. I suspect that we are in the former case... See the comments on the PR, for more suggestions on tests to add. 

And obviously, we nedd that for Hed2. Head3, HeadMN*, HeadNN{a,b,c}*, ....